specification/iotcentral/data-plane/Microsoft.IoTCentral/stable/2022-05-31/iotcentral.json

```diff
@@ -0,0 +1,3565 @@
+      "type": "object",
+      "properties": {
+        "@id": {
+          "description": "Unique ID of the device template.",
+          "type": "string",
+          "readOnly": true
+        },
+        "@type": {
+          "description": "The JSON-LD types of this device template.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "etag": {
+          "description": "ETag used to prevent conflict in device template updates.",
+          "type": "string"
+        },
+        "displayName": {
+          "description": "Display name of the device template.",
+          "type": "string"
+        },
+        "description": {
+          "description": "Detailed description of the device template.",
+          "type": "string"
+        },
+        "capabilityModel": {
+          "description": "The capability model utilized by this device template.",
+          "type": "object"
+        }
+      },
+      "required": [
+        "@type",
+        "capabilityModel"
+      ]
+    },
+    "DeviceTemplateCollection": {
+      "description": "The paged results of device templates.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The collection of device templates.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DeviceTemplate"
+          }
+        },
+        "nextLink": {
+          "description": "URL to get the next page of device templates.",
+          "type": "string"
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "EmailUser": {
+      "description": "The email user definition.",
+      "x-ms-discriminator-value": "email",
+      "allOf": [
+        {
+          "$ref": "#/definitions/User"
+        }
+      ],
+      "type": "object",
+      "properties": {
+        "email": {
+          "description": "Email address of the user.",
+          "type": "string"
+        }
+      },
+      "required": [
+        "email"
+      ]
+    },
+    "Error": {
+      "description": "The response error definition",
+      "type": "object",
+      "properties": {
+        "error": {
+          "description": "Error details for current request.",
+          "$ref": "#/definitions/ErrorDetails"
+        }
+      },
+      "required": [
+        "error"
+      ]
+    },
+    "ErrorDetails": {
+      "description": "The detail information of the error.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "Error code."
+        },
+        "message": {
+          "type": "string",
+          "description": "Error message details."
+        },
+        "requestId": {
+          "type": "string",
+          "description": "Correlation Id for current request."
+        },
+        "time": {
+          "type": "string",
+          "description": "The time that error request failed."
+        }
```

Comment:
why is this `time` property does not have a format? rfc1123 or rfc3339?

Reply:
rfc1123, was used for customer report their CSS ticket when API had some issues, same as "Date" header value.

Reply:
Then I think you need to add a `format: date-time-rfc1123` here.

Reply:
Added as suggest.